File: scala-play-server/app/model/ResponseFares.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ResponseFares.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2020-07-14T11:29:17.777Z[Etc/UTC]")
case class ResponseFares(
  breakdown: List[ResponseFaresBreakdownItem],
  ticketsTotal: List[ResponseFareTicket]
)

object ResponseFares {
  implicit lazy val responseFaresJsonFormat: Format[ResponseFares] = Json.format[ResponseFares]
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseFares.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.ResponseFareTicket
open traveltimeplatform.Model.ResponseFaresBreakdownItem

module ResponseFares = 

  //#region ResponseFares


  type ResponseFares = {
    Breakdown : ResponseFaresBreakdownItem[];
    TicketsTotal : ResponseFareTicket[];
  }
  //#endregion
  